test/cat.jl

```diff
@@ -57,11 +49,24 @@ module TestCat
     @testset "hcat ::Vectors" begin
         df = DataFrame()
         DataFrames.hcat!(df, CategoricalVector{Union{Int, Missing}}(1:10))
+        @test df[1] == collect(1:10)
         @test df[1] == CategoricalVector(1:10)
```

Comment:
This line is redundant with the previous one (or vice-versa).

Reply:
fixed - these were old tests.